interface/client/templates/elements/networkIndicator.js

```diff
@@ -0,0 +1,82 @@
+/**
+Template Controllers
+
+@module Templates
+*/
+
+/**
+The networkIndicator template
+
+@class [template] elements_networkIndicator
+@constructor
+*/
+
+
+
+/**
+Check network type.
+
+@method checkNetworkType
+*/
+var checkNetworkType = function(template) {
+    console.trace('Check network type...');
+
+    try {
```

Comment:
Why the try block? It should return an error in the callback if something is wrong